### DISCUSSION TITLE
What is the tmp directory under the BE data directory for?

### DISCUSSION THREAD

**Discussion by eleven-f** [apache/doris#41028] [A - General / Q&A]
Version: 2.1.5
What is the tmp directory under the BE data directory for?
We have a 6-node (*4 disk) cluster. There is a tmp directory under the data directory of one of the BE nodes, which occupies a lot of disk space (200G * 4 disks). What is this directory for? How is the data in it generated? How to optimize this usage?

版本：2.1.5
BE 数据目录下的 tmp 目录是干什么的？
我们有个6节点（*4盘）的集群，其中一个 BE 节点的数据目录下有个tmp目录，占用了大量磁盘空间(200G * 4盘)，请问这个目录是干什么的？ 里面的数据因何产生的？

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去 [Doris 中文技术论坛](https://doris-forum.org.cn)。

**Comment by leowakeup** [comment]
这个问题找到答案了吗？

**Reply by eleven-f** [reply]
没有
